components/tools/OmeroJava/test/integration/gateway/TablesFacilityTest.java

```diff
@@ -276,5 +281,46 @@ public void testThreshold() throws Exception {
         Assert.assertEquals(td.getData()[0].length,
                 TablesFacility.DEFAULT_MAX_ROWS_TO_FETCH);
     }
+    
+    @Test(dependsOnMethods = { "testThreshold" })
+    public void testUpdateTable() throws Exception {
```

Comment:
``cols= td.getColumns()`` same for data 
since the various will have an effect on performance

Reply:
I'll leave @dominikl to respond, but I think these are all local invocations since it's `omero.gateway.model.TableData` and not `omero.api.TablesPrx`.

Reply:
in that case no need to make changes
I got lost in the PR review

Reply:
Yes, just accesses the `columns` array in the `TableData` object.